core/tests/test_utils.py

```diff
@@ -254,6 +340,24 @@ def post_json(self, url, payload, csrf_token=None, expect_errors=False,
     def _send_post_request(
             self, app, url, data, expect_errors=False, expected_status_int=200,
             upload_files=None, headers=None):
+        """Make a post api call.
+
+        Args:
+            app: application that is being tested
+            url: str. The URL to make a POST request to.
+            data: dict (json). data sent to the server to make
```

Comment:
Same comments as in previous method re removing "(json)", starting with a capital letter, indenting continuation lines by 4, etc.